registrations/models.py

```diff
@@ -587,17 +587,23 @@ def get_registration_message(self, subject, cleaned_body, plain_text_body):
             self.service_language
         )
 
-        email_variables = {
-            "body": cleaned_body,
-            "linked_events_ui_locale": linked_events_ui_locale,
-            "linked_events_ui_url": settings.LINKED_EVENTS_UI_URL,
-            "linked_registrations_ui_locale": linked_registrations_ui_locale,
-            "linked_registrations_ui_url": settings.LINKED_REGISTRATIONS_UI_URL,
-            "registration_id": self.registration_id,
-            "signup": self,
-        }
-
         with override(linked_registrations_ui_locale, deactivate=True):
+            event_name = self.registration.event.name
+            event_type_id = self.registration.event.type_id
+            signup_edit_url = get_signup_edit_url(self, linked_registrations_ui_locale)
+
+            email_variables = {
+                "body": cleaned_body,
+                "event": event_name,
```

Comment:
Change this (and the related email template) to:
```suggestion
                "event_name": event_name,
```
This is to make it explicit to the reader that it's an event's name, not an event instance.

Reply:
Renamed the variable. I did the same change to other places where the same naming was used.